gravitee-apim-console-webui/src/management/application/details/general/application-general.component.ts

```diff
@@ -164,6 +183,9 @@ export class ApplicationGeneralComponent implements OnInit {
               oauth: {
                 ...this.initialApplication.settings.oauth,
                 ...this.applicationForm.getRawValue().OpenIDForm,
+                additional_client_metadata: this.applicationForm
+                  .getRawValue()
+                  .OpenIDForm.additional_client_metadata.reduce((acc, { key, value }) => ({ ...acc, [key]: value }), {}),
```

Comment:
WDYT to create same as creation `toMetadataDictionary` (or `toOAuthAdditionalClientMetadata()`) ? and add `?` to handle case if additional_client_metadata is null or undefined ?